packages/core/src/utils/getAction.ts

```diff
@@ -27,12 +27,16 @@ export function getAction<
 >(
   client: client,
   actionFn: (_: any, parameters: parameters) => returnType,
-  // Some minifiers drop `Function.prototype.name`, meaning that `action.name`
-  // will not work. For that case, the consumer needs to pass the name explicitly.
+  // Some minifiers drop `Function.prototype.name`, or replace it with short letters,
+  // meaning that `actionFn.name` will not always work. For that case, the consumer
+  // needs to pass the name explicitly.
   name: keyof PublicActions | keyof WalletActions,
 ): (parameters: parameters) => returnType {
-  const action = client[actionFn.name ?? name]
```

Comment:
in my case `actionFn.name` was not undefined but had the minified letter `w`, so it always used the fallback function, even though the function on the wallet client existed.